qpc/cred/add.py

```diff
@@ -64,8 +64,14 @@ def __init__(self, subparsers):
                                  action='store_true',
                                  help=_(messages.CRED_SSH_PSPH_HELP))
         self.parser.add_argument('--become_method', dest='become_method',
-                                 choices=['sudo', 'su', 'pbrun', 'pfexec',
-                                          'doas', 'dzdo', 'ksu', 'runas'],
+                                 choices=[credential.BECOME_SUDO,
```

Comment:
Could setup the list of choices as constant also to avoid differences between add/edit.